posts/gsoc-2020-blog-3.md

```diff
@@ -0,0 +1,164 @@
+---
+title: GSoC 2020 Blog Post 3
+date: 2020-07-26 16:20:00 UTC-04:00
+slug: gsoc-2020-blog-3
+tags: GSoC 2020
+description: Developer-oriented detail about how ReactorNet's time integration methods actually work.
+type: text
+author: Paul Blum
+---
+
+## _How Does Cantera's Reactor Network Time Integration Feature Work?_
+
+There's a great description of the science behind Cantera's reactor network simulation capabilities available on the Cantera website, [here](https://cantera.org/science/reactors.html). This post will go into more developer-oriented detail about how `ReactorNet`'s [time integration methods](https://cantera.org/science/reactors.html#time-integration) actually work.
+
+A `ReactorNet` object doesn't perform time integration on its own. It compiles a system of ODE's based on the combined governing equations of all contained `Reactor`s, which is then passed off to an `Integrator` object for solution. What is an `Integrator`?
```

Comment:
Compiles here might imply some sort of compiler process, which isn't used, everything is done at runtime. Maybe creates or generates?

In the next paragraph, it's better to avoid words like easy and simple in tutorials like this. It won't be simple to everyone how to include a header file!